api/grpcserver/http_server.go

```diff
@@ -109,10 +112,22 @@ func (s *JSONHTTPServer) StartService(
 		}
 	}
 
+	var c *cors.Cors
 	// enable cors
-	c := cors.New(cors.Options{
-		AllowedOrigins: s.origins,
-	})
+	if s.corsEverywhere || true {
```

Comment:
that `|| true` is added to check if it's working regardless of the setup. And it currently does not. Need to check why exactly.

Reply:
fixed